### ISSUE TITLE
Issue if file paths in git repo have spaces in name

### ISSUE THREAD

**Issue by BGraco** [AlDanial/cloc#257]
On OS X 10.12.6 running cloc 1.74 installed via brew.
Running cloc --git --diff on a git repo which has files with spaces will cause an error:
```
user:smartplot$ git diff --stat 5e2716b6abda94982d7ec0f49b8cca2f8c7cd44b 3c5f36c9fa8ade75c0b93c27b475c98533ad547e
 .gitignore                                            |     9 +-
 dist/make installer command.txt                       |     1 +
 images/icon_source.pdn                                |   Bin 0 -> 44059 bytes
 images/icon_source.png                                |   Bin 0 -> 18822 bytes
.....
 134 files changed, 38203 insertions(+), 29907 deletions(-)

user:smartplot$ cloc --git --diff 5e2716b6abda94982d7ec0f49b8cca2f8c7cd44b 3c5f36c9fa8ade75c0b93c27b475c98533ad547e
fatal: pathspec 'dist/make' did not match any files
Failed to create tarfile of files from git. at /usr/local/Cellar/cloc/1.74/libexec/bin/cloc line 3811.
```

**Comment by AlDanial** [comment]
Let me know if the fix in commit 41bcab8 works for you.

**Comment by BGraco** [comment]
That works great. Thank you.
